pkg/util/iptables/iptables.go

```diff
@@ -120,6 +120,56 @@ const MinCheckVersion = "1.4.11"
 const MinWaitVersion = "1.4.20"
 const MinWait2Version = "1.4.22"
 
+// no-op implemenatation of Interface
+type fake struct{}
+
+func NewFake() *fake {
```

Comment:
Yes, but I guess need to either copy the interface definition to that package (and everyone updating it will need to remember to update both), or just import everything (i.e. both 'real' and 'fake' packages) whenever I use fake implementation. It's not terrible, but I'm not sure if it's better than making a binary a little bit bigger.